theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -36,10 +36,12 @@ let fundingAgenciesApp = new Vue({
     crossreffApiDown: false, // if we are having trouble reaching the crossref api
     removeCharsFromQuery: ["."], // characters to be removed from search
     filteredWords: [],
+    ROR_API_URL: "https://api.ror.org/v2/organizations"
   },
   mounted() {
-    if (this.resourceMode === 'Edit' && (this.selfAccessLevel === 'owner' || this.selfAccessLevel === 'editor') && this.resPublished) {
-      this.checkFunderNamesExistInCrossref(this.fundingAgencies);
+    if (this.resourceMode === 'Edit' && (this.selfAccessLevel === 'owner' || this.selfAccessLevel === 'editor') ) {
+      // this.checkFunderNamesExistInCrossref(this.fundingAgencies);
```

Comment:
remove this commented out line
and we can get rid of the `checkFunderNamesExistInCrossref()` function